src/Raven.Server/Utils/RavenCli.cs

```diff
@@ -0,0 +1,706 @@
+            Console.ResetColor();
+            Console.Write("Garbage Collecting... ");
+            Console.Out.Flush();
+
+            switch (genNum)
+            {
+                case 0:
+                    GC.Collect(0);
+                    break;
+                case 1:
+                    GC.Collect(1);
+                    break;
+                case 2:
+                    GC.Collect(GC.MaxGeneration);
+                    break;
+                default:
+                    WriteError("Invalid argument passed to GC. Can be 0, 1 or 2");
+                    return false;
+            }
+
+            GC.WaitForPendingFinalizers();
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Collected.");
+            Console.ResetColor();
+            Console.Write("After collecting, managed memory used:  ");
+            Console.ForegroundColor = ConsoleColor.Cyan;
+            Console.WriteLine(new Size(GC.GetTotalMemory(false), SizeUnit.Bytes));
+            Console.ResetColor();
+            Console.Out.Flush();
+
+            return true;
+        }
+
+        private static bool CommandLog(List<string> args)
+        {
+            switch (args.First())
+            {
+                case "on":
+                    LoggingSource.Instance.EnableConsoleLogging();
+                    LoggingSource.Instance.SetupLogMode(LogMode.Information, Path.Combine(AppContext.BaseDirectory, _server.Configuration.Logs.Path));
+                    break;
+                case "off":
+                    LoggingSource.Instance.DisableConsoleLogging();
+                    LoggingSource.Instance.SetupLogMode(LogMode.None, Path.Combine(AppContext.BaseDirectory, _server.Configuration.Logs.Path));
+                    break;
+                case "http-off":
+                    RavenServerStartup.SkipHttpLogging = true;
+                    goto case "on";
+            }
+
+            return true;
+        }
+
+        private static bool CommandClear(List<string> args)
+        {
+            Console.Clear();
+            Console.Out.Flush();
+            return true;
+        }
+
+        private static bool CommandInfo(List<string> args)
+        {
+            var memoryInfo = MemoryInformation.GetMemoryInfo();
+            Console.ForegroundColor = ConsoleColor.Cyan;
+            Console.WriteLine(" Build {0}, Version {1}, SemVer {2}, Commit {3}\r\n PID {4}, {5} bits, {6} Cores, Arch: {9}\r\n {7} Physical Memory, {8} Available Memory",
+                ServerVersion.Build, ServerVersion.Version, ServerVersion.FullVersion, ServerVersion.CommitHash, Process.GetCurrentProcess().Id,
+                IntPtr.Size * 8, ProcessorInfo.ProcessorCount, memoryInfo.TotalPhysicalMemory, memoryInfo.AvailableMemory, RuntimeInformation.OSArchitecture);
+
+            var bitsNum = IntPtr.Size * 8;
+            if (bitsNum == 64 && _server.Configuration.Storage.ForceUsing32BitsPager)
```

Comment:
we write x64 or x86 always. This is a unique place where running on 64bits enforcing 32bits. And it is a rare one. I am not sure I would want to see "Enforce 32 bit: No" all the time